### DISCUSSION TITLE
Discord/telegram?

### DISCUSSION THREAD

**Discussion by Charlesverde** [KilledByAPixel/LittleJS#32] [General]
Hi, I was wondering if there is a discord or other place to discuss projects?

I'm looking to start a project with this framework but still pretty new to game development in general and would love to bounce some questions and discuss issues with others.

**Comment by KilledByAPixel** [comment]
Hi, I just started up a discord, may take some time for people to show up though. I expect this engine will get a lot of use when JS13k comes around in august.

https://discord.gg/zb7hcGkyZe (updated link)

**Reply by dshepsis** [reply]
Hey, I noticed this invite is expired. Does this server still exist?

**Reply by KilledByAPixel** [reply]
hey, here is a new link, you are welcome to join... https://discord.gg/zb7hcGkyZe
